spec/index.bs

```diff
@@ -2312,6 +2315,11 @@ interface GPU {
                     {{GPUTextureFormat/"bgra8unorm"}}, depending on which format is optimal for
                     displaying WebGPU canvases on this system.
         </div>
+
+    : <dfn dfn-type=attribute>shadingSoftwareExtensions</dfn>
```

Comment:
Using "WGSL" seemed to jargon-y here.
Also it would be an acronym in camel-case, and that would make it wgslSoftwareExtensions which looks weird.

Choosing between "shader" and "shading" because WGSL is WebGPU "Shading" Language (per Apple preference).

Reply:
Personally I prefer "wgsl" for being direct about what it's talking about, but forgot to bring this up with editors, so can keep as is for now